server/webapp/WEB-INF/rails/webpack/views/pages/new_plugins/index.scss.d.ts

```diff
@@ -2,6 +2,7 @@
 // Please do not change this file!
 export const pluginHeader: string;
 export const pluginIcon: string;
+export const pluginName: string;
```

Comment:
Not required. Already done as part of https://github.com/gocd/gocd/commit/afa4c88794477caccc98c5765cbfb54c015edf27 commit